frontend/src/scenes/onboarding/sdks/SDKSnippet.tsx

```diff
@@ -1,7 +1,26 @@
-import { SDK } from '~/types'
-import { productAvailableSDKs } from './sdksLogic'
+import { SDK, SDKInstructionsMap } from '~/types'
+import { IconOpenInNew } from 'lib/lemon-ui/icons'
+import { Link } from 'lib/lemon-ui/Link'
 
-export const SDKSnippet = ({ sdk, productKey }: { sdk: SDK; productKey: string }): JSX.Element => {
-    const SDKInstructions = productAvailableSDKs[productKey][sdk.key]
-    return SDKInstructions ? <SDKInstructions /> : <></>
+export const SDKSnippet = ({
+    sdk,
+    sdkInstructionMap,
+}: {
+    sdk: SDK
+    sdkInstructionMap: SDKInstructionsMap
+}): JSX.Element => {
+    const SDKInstructions = sdkInstructionMap[sdk.key]
+    return SDKInstructions ? (
```

Comment:
I'm generally against components that conditionally render but maybe that's a personal preference. In this case it's called from
```
<div className="shrink min-w-8">
    <SDKSnippet sdk={selectedSDK} sdkInstructionMap={sdkInstructionMap} />
</div>
```
so the width will be applied even if nothing is rendered. Usually I find it easier to let the caller decide if the component should be rendered:
```
{!!sdkInstructionMap[selectedSDK] && (
  <div className="shrink min-w-8">
      <SDKSnippet sdk={selectedSDK} sdkInstructions={sdkInstructions[selectedSDK]} />
  </div>
)}
```

Then you shouldn't need to check the instructions map within this component

Reply:
Good call, did as you suggest 👍